crates/services/upgradable-executor/src/config.rs

```diff
@@ -1,11 +1,14 @@
 use fuel_core_executor::executor::ExecutionOptions;
+use fuel_core_types::blockchain::header::StateTransitionBytecodeVersion;
 
 #[derive(Clone, Debug, Default)]
 pub struct Config {
     /// Print execution backtraces if transaction execution reverts.
     pub backtrace: bool,
     /// Default mode for utxo_validation
     pub utxo_validation_default: bool,
+    /// The version of the native executor to determine usage of native vs WASM executor.
```

Comment:
> to determine usage of native vs WASM executor

Meaning that if this is `None`, it will use Wasm? How does it determine which one to use?